examples/datacenters/aws-k8s/aws-lb-controller/index.ts

```diff
@@ -0,0 +1,62 @@
+import { local } from "@pulumi/command";
+import * as kubernetes from "@pulumi/kubernetes";
+import * as pulumi from "@pulumi/pulumi";
+
+const config = new pulumi.Config();
+const name = config.require("name");
+const clusterName = config.require("clusterName");
+const accountId = config.require("accountId")
+
+const awsConfig = config.requireObject("aws") as {
+  accessKey: string,
+  secretKey: string,
+  region: string
+};
+const saName = 'aws-load-balancer-controller';
+
+// Create this policy for the AWS Load Balancer if it doesn't already exist
+const iam_policy = new local.Command("iam_policy", {
+  create: "aws iam create-policy --policy-name AWSLoadBalancerControllerIAMPolicy --policy-document file://iam_policy.json || true",
+  environment: {
+    "AWS_ACCESS_KEY_ID": awsConfig.accessKey,
+    "AWS_SECRET_ACCESS_KEY": awsConfig.secretKey,
+    "AWS_DEFAULT_REGION": awsConfig.region,
+  }
+});
```

Comment:
This is interesting. Why via a command instead of the AWS provider? Won't this leave the policy dangling when the datacenter gets cleaned up?

Reply:
Mostly because the `eksctl` command below needs the same env variables and needs the AWS CLI installed already, so it was easier to just run this as a command vs. add the AWS provider (where otherwise in this file we're just using the Kubernetes provider).

The `eksctl` command can be done with Pulumi/Terraform but it's kind of comple - the CLI tool is the recommended way and makes it super simple simple so I opted to do that. 

I purposely wasn't deleting this policy when cleaning up because the policy only needs to exist once for your AWS DCs and can be re-used - technically I could give it a unique name and make/delete it for each DC, but a policy alone doesn't actually do anything so it's kind of harmless to just reuse the same one.